packages/serve/index.ts

```diff
@@ -94,60 +93,64 @@ function serve() {
 		process.stdout.write(
 			"\n" +
 				chalk.bold(
-					"✖ We didn't find any webpack-dev-server dependency in your project,"
+					"✖ We didn't find any webpack-dev-server dependency in your project,",
 				) +
 				"\n" +
 				chalk.bold.green("  'webpack serve'") +
 				" " +
 				chalk.bold("requires you to have it installed ") +
-				"\n\n"
+				"\n\n",
 		);
 		return inquirer
 			.prompt([
 				{
-					type: "confirm",
-					name: "confirmDevserver",
+					default: "Y",
 					message: "Do you want to install it? (default: Y)",
-					default: "Y"
-				}
+					name: "confirmDevserver",
+					type: "confirm",
+				},
 			])
-			.then(answer => {
-				if (answer["confirmDevserver"]) {
+			.then((answer: {
+				confirmDevserver: boolean,
+			}) => {
+				if (answer.confirmDevserver) {
 					return inquirer
 						.prompt(
 							List(
 								"confirmDepType",
 								"What kind of dependency do you want it to be under? (default: devDependency)",
-								["devDependency", "optionalDependency", "dependency"]
-							)
+								["devDependency", "optionalDependency", "dependency"],
+							),
 						)
-						.then(depTypeAns => {
-							const packager = getRootPathModule("package-lock.json")
+						.then((depTypeAns: {
+							confirmDepType: string;
+						}) => {
```

Comment:
void for all empty funcs below this line

Reply:
Please check above comment.

Reply:
`_?: void` syntax